web_ui/src/providers/dataset-import-to-existing-project-provider/dataset-import-to-existing-project-provider.component.tsx

```diff
@@ -134,6 +135,12 @@ export const DatasetImportToExistingProjectProvider = ({ children }: DatasetImpo
             { workspaceId, organizationId, fileId: String(activeDatasetImport.uploadId) },
             {
                 onSuccess: () => deleteLsDatasetImport(activeDatasetImport.id),
+                onError: (error) => {
+                    if (error.response?.status === HttpStatusCode.NotFound) {
+                        deleteLsDatasetImport(activeDatasetImport.id);
+                    }
+                    addNotification({ message: error.message, type: NOTIFICATION_TYPE.ERROR });
```

Comment:
Do we need to show this notification when the dataset import no longer exists? I feel like it might be fine to just acknowledge that it has been removed?